graylog2-server/src/main/java/org/graylog2/periodical/ClusterHealthCheckThread.java

```diff
@@ -103,7 +111,7 @@ public boolean startOnThisNode() {
     public int getInitialDelaySeconds() {
         // Wait some time until all inputs have been started otherwise this will trigger a notification on every
         // startup of the server.
-        return 120;
+        return 1;
```

Comment:
Why did you reduce this to `1`?

Reply:
Ack, accidental commit. Thanks for seeing this.